OgreMain/src/OgrePlatformInformation.cpp

```diff
@@ -639,7 +639,7 @@ namespace Ogre {
     {
         uint32 numLogicalCores = 0;
 
-#if OGRE_PLATFORM == OGRE_PLATFORM_LINUX || OGRE_PLATFORM == OGRE_PLATFORM_ANDROID
+#if OGRE_PLATFORM == OGRE_PLATFORM_LINUX || OGRE_PLATFORM == OGRE_PLATFORM_ANDROID || OGRE_PLATFORM_FREEBSD
```

Comment:
This line here is causing trouble.

`OGRE_PLATFORM_FREEBSD` should be `OGRE_PLATFORM == OGRE_PLATFORM_FREEBSD`

Reply:
Fixed it in 4d3aa2a.